CMSgov/design-system
# [WNGDS-2978] Add customizable `arialabel` prop to `Tabs` component.

## Summary

- Added `ariaLabel` to the `Tabs` component interface and introduced `tabsAriaLabel` as an alias for `ariaLabel` specifically for use in the web component (`ds-tabs`). If neither `ariaLabel` nor the alias attribute `tabs-aria-label` is provided, a default value of "Tabs Navigation" will be used for the aria-label in the HTML markup.
- Updated the `ds-tabs` wrapper component to correctly map the `tabs-aria-label` attribute to the `ariaLabel` prop.

## How to test

1.  Run `yarn storybook`
2.  Inspect the rendered `Tabs` component in the chrome devTools elements tab to verify that the `aria-label` attribute is present in the HTML markup.
3. Inspect the rendered `ds-tabs` component in the chrome devTools elements tab. Use the controls to add a value to `tabs-aria-label` and verify that it is correctly passed to the `tablist` element as `aria-label` without duplication.

## Checklist
